idl/arm_batch.pro

```diff
@@ -270,7 +272,11 @@ arm_movies, today_dir, date_struct
 ; Execute the forecast last as its prone to crashing
 
 if ( summary[ 0 ] ne 'No data' ) then arm_forecast, output_path=today_dir, date_struct, summary
-  
+
+yest=anytim(date_struct.date_dir)-findgen(2)*3600.*24.
```

Comment:
Can these variables not be assigned within your chimera script? Best to pass date struct to your script as input and define these variables within